pkgs/development/compilers/pawn/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchzip, pkgs }:
+
+stdenv.mkDerivation rec {
+  pname = "pawn";
+  version = "4.0.5749";
+
+  src = fetchzip {
+    url = "https://www.compuphase.com/${pname}/${pname}-${version}.zip";
+    sha256 = "1qj2n0xql2bmgvng7q7xf7113bgi2kqgaq9jp0dmys3nc3vm123f";
+    stripRoot = false;
+  };
+  buildInputs = with pkgs; [ cmake gcc_multi ];
```

Comment:
Why not to declare them on function arguments?

Also, `cmake` usually goes in `nativeBuildInputs`.